src/SprykerEco/Zed/Computop/Business/Oms/Command/Manager/CancelManager.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Computop\Business\Oms\Command\Manager;
+
+use Generated\Shared\Transfer\OrderTransfer;
+use Orm\Zed\Sales\Persistence\SpySalesOrderItem;
+use Propel\Runtime\ActiveQuery\Criteria;
+
+class CancelManager extends AbstractManager implements CancelManagerInterface
+{
+    /**
+     * @param array $orderItems
+     *
+     * @return array
+     */
+    public function changeComputopItemsStatus(array $orderItems)
+    {
+        $this->handleDatabaseTransaction(function () use ($orderItems) {
+            foreach ($orderItems as $orderItem) {
+                $this->changeStatus($orderItem);
+            }
+        });
+
+        return [];
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     *
+     * @return array
+     */
+    public function getCanceledItems(OrderTransfer $orderTransfer)
+    {
+        return $this
+            ->queryContainer
+            ->getSpySalesOrderItemsById($orderTransfer->getIdSalesOrder())
```

Comment:
Move this query to query the container, something like `>getCanceledSalesOrderItemsById`